assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -1713,5 +1713,34 @@ describe("DiversionPage", () => {
         screen.getByRole("combobox", { name: "Choose route" })
       ).toHaveAccessibleErrorMessage("Select a route to continue.")
     })
+
+    test("while on this panel, route is not interactive", async () => {
```

Comment:
Awesome sauce!

This test is a perfect example of a test that will be a lot easier to write and work with once we build some better scaffolding for detour-related tests.

Thanks for adding it!